### PR TITLE
Secure cookies

### PR DESCRIPTION
**Pull request by monikasulik**
FYI, I've not noticed the csrf setting actually doing anything (I've not seen any csrf cookies being set). I presume this is because we pretty much always have a session when filling out forms.

### CODE REVIEW

File: ui/settings.py [hunk 1]
```diff
@@ -135,6 +135,10 @@
 }
 
 SESSION_ENGINE = "django.contrib.sessions.backends.signed_cookies"
+SESSION_COOKIE_SECURE = True if (
+    os.getenv('UI_SESSION_COOKIE_SECURE') == 'true') else False
```
Comment (richtier) [inline ui/settings.py]: could be simplified to `SESSION_COOKIE_SECURE = os.getenv('UI_SESSION_COOKIE_SECURE') == 'true'` :)

Comment (richtier) [inline ui/settings.py]: can we call the variable `SESSION_COOKIE_SECURE` without the prefix?

**Comment by konradko** [comment]
@monikasulik @richtier if we merge this then we have to add the rewrite to HTTPS from HTTP as well

**Comment by monikasulik** [comment]
ah, I already merged the sso side of this change: https://github.com/uktrade/directory-sso/pull/40
That said, both this and the https rewrite is something that I don't think we can go live without anyway.

**Comment by konradko** [comment]
@monikasulik fine to merge actually, just have to advise everyone to use HTTPS, otherwise won't work

**Pull request merged**